.expeditor/config.yml

```diff
@@ -15,17 +15,18 @@ github:
   major_bump_labels:
     - "Expeditor: Bump Version Major"
 
-# https://expeditor.chef.io/docs/getting-started/subscriptions/#merge-actions
-# The `merge_actions` key is a shortcut for the
-# `pull_request_merged:<GITHUB_REPO>:<RELEASE_BRANCH>:*` subscription.
-merge_actions:
-  - built_in:bump_version:
-      ignore_labels:
-        - "Expeditor: Skip Version Bump"
-        - "Expeditor: Skip All"
-  - bash:.expeditor/update_version.sh:
-      only_if: built_in:bump_version
-  - built_in:update_changelog:
-      ignore_labels:
-        - "Expeditor: Skip Changelog"
-        - "Expeditor: Skip All"
+subscriptions:
+  # The `merge_actions` key is a shortcut for the
```

Comment:
Since we have removed the `merge_actions` key, it doesn't make sense to reference it (or the fact that it maps to the subscription we've introduced). Lines 19 and 20 can be removed.

Reply:
removed them